manage-data/lifecycle/index-lifecycle-management/tutorial-time-series-with-data-streams.md

```diff
@@ -30,22 +30,21 @@ When you enable {{ilm}} for {{beats}} or the {{ls}} {{es}} output plugin, lifecy
 To set up a data stream, first create an index template to specify the lifecycle policy. Because the template is for a data stream, it must also include a `data_stream` definition.
 
 For example, you might create a template named `timeseries_template` and use it for a future data stream named `timeseries`.
+To configure {{ilm-init}} to manage the data stream, you specify the name of the lifecycle policy that you want to apply to the data stream with the `index.lifecycle.name` setting.
 
-To enable {{ilm-init}} to manage the data stream, the template configures one {{ilm-init}} setting:
-
-* `index.lifecycle.name` specifies the name of the lifecycle policy that you want to apply to the data stream.
-
-Use the {{kib}} **Create template** wizard to add a template or the [Create or update index template](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-index-template) API to add a template and apply the lifecycle policy to indices matching the template.
+Use the {{kib}} **Create template** wizard to add a template or the [Create or update index template](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-index-template) API to add a template and apply the lifecycle policy to indices matching the template. The wizard invokes the create or update index template API to create the index template with the options you specify.
```

Comment:
Just wondering: does it help users to know that the UI invokes the API? To me that's kind of in the "how the code works" category, and I think it would be true of all of the UI tasks that we document.